packages/ui/src/hooks/useDisplayLoader.tsx

```diff
@@ -15,48 +14,39 @@ export const useDisplayLoader = () => {
 
   if (!isWatchAddressInitialized || !isLocalStorageSetupDone) {
     return (
-      <LoaderBoxStyled data-cy="loader-initialization">
-        <CircularProgress />
-        Initialization...
-      </LoaderBoxStyled>
+      <LoadingBox
+        message="Initialization..."
+        dataCy="loader-initialization"
```

Comment:
and these will then change too
```suggestion
        testId="initialization"
```

Reply:
Thank you for your suggestions! Now it looks even better!